definitions/bulk-action/json/CaseEvent.json

```diff
@@ -19,6 +19,8 @@
     "DisplayOrder": 2,
     "PreConditionState(s)": "Created",
     "PostConditionState": "ScheduledForListing",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/bulk/schedule",
```

Comment:
Does this exist yet?

Reply:
Not yet, but since ScheduleForListing won't go live without this functionality, I think its safe to merge. Built in https://github.com/hmcts/div-case-orchestration-service/pull/285.